### PR TITLE
Pick A Buyer redesign

### PR DESCRIPTION
**Pull request by pimms**
# Why?

We're implementing the pick-a-buyer flow as originally envisioned, now that we have complete ownership of Messaging.

# What?

 - Renamed `ReviewView` to `BuyerPickerView` _(and all associated types similarly)_
 - Removed button & label at the bottom — tapping the cells will kickstart the next step
 - General refactoring & removal of unused properties

# Show me

| Before | After |
| --- | --- |
| image | image |

### CODE REVIEW

File: Sources/Fullscreen/Review/BuyerPickerView.swift [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        guard let selectedUser = model?.profiles[indexPath.row] else { return }
+        delegate?.buyerPickerView(self, didSelect: selectedUser)
+    }
+}
+
+// MARK: - ReviewProfileCellDelegate
+
+extension BuyerPickerView: BuyerPickerCellDelegate {
+    func buyerPickerCell(_ reviewProfileCell: BuyerPickerProfileCell, loadImageForModel model: BuyerPickerProfileModel, imageWidth: CGFloat, completion: @escaping ((UIImage?) -> Void)) -> UIImage? {
+        return delegate?.buyerPickerView(self, loadImageForModel: model, imageWidth: imageWidth, completion: completion)
```
Comment (ag12) [inline Sources/Fullscreen/Review/BuyerPickerView.swift]: deilig.

Comment (pimms) [reply Sources/Fullscreen/Review/BuyerPickerView.swift]: lol jepp, fikser denne nå

**Comment by pimms** [comment]
This PR is quite achy-breaky, so I won't merge it until we've completed the integration in `ios-app`.

**Comment by ag12** [comment]
GJ

**Pull request merged**